orderer/common/server/main.go

```diff
@@ -845,6 +889,48 @@ func initializeEtcdraftConsenter(consenters map[string]consensus.Consenter, conf
 	consenters["etcdraft"] = raftConsenter
 }
 
+func initializeSmartBFTConsenter(
+	signer identity.SignerSerializer,
+	dpmr *DynamicPolicyManagerRegistry,
+	consenters map[string]consensus.Consenter,
+	conf *localconfig.TopLevel,
+	lf blockledger.Factory,
+	clusterDialer *cluster.PredicateDialer,
+	bootstrapBlock *cb.Block,
+	ri *onboarding.ReplicationInitiator,
+	srvConf comm.ServerConfig,
+	srv *comm.GRPCServer,
+	registrar *multichannel.Registrar,
+	metricsProvider metrics.Provider,
+	bccsp bccsp.BCCSP,
+) *smartbft.Consenter {
+	fmt.Println("DEBUG-initializeSmartBFTConsenter called")
+	systemChannelName, err := protoutil.GetChannelIDFromBlock(bootstrapBlock)
+	if err != nil {
+		logger.Panicf("Failed extracting system channel name from bootstrap block: %v", err)
+	}
+	systemLedger, err := lf.GetOrCreate(systemChannelName)
+	if err != nil {
+		logger.Panicf("Failed obtaining system channel (%s) ledger: %v", systemChannelName, err)
+	}
+	getConfigBlock := func() *cb.Block {
+		return multichannel.ConfigBlockOrPanic(systemLedger)
+	}
+	icr := onboarding.NewInactiveChainReplicator(ri, getConfigBlock, ri.RegisterChain, conf.General.Cluster.ReplicationBackgroundRefreshInterval)
```

Comment:
we don't have this inactive chain replication thingy anymore, because we are using channel participation.

Reply:
updated it